nss-day-cohort-48/giffygram-jumpstation
# Send Direct Messages

#### Changes Made

1. Modified file `MessageForm.js` to include `the ability to open and close the direct message form as well as added the ability to send a direct message.` 
   ​
####IMPORTANT:
In your giffygram.json file you must manually add an empty  messages array.

i.e. "messages": []

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout direct-messages
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   >Click the "Send A Message" button and fill out the form that appears. 
   >Click save then verify there is a new entry in your messages array in the giffygram.json database. 
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
